scss/grid/_grid.scss

```diff
@@ -30,6 +30,11 @@ $grid-column-align-edge: true !default;
 /// @type Number
 $block-grid-max: 8 !default;
 
+/// Selector used for an alias of column (with @extend). If `false`, no alias is created.
+/// @since 6.4.0
```

Comment:
According to the docs:

> Describes the version at which the documented item has been implemented _**or updated**_.

So we describe a feature and tell from what version it is like that. It can be helpful to understand why new features described in the docs are not available in an older version, but it is confusing for updated features: is it new, it is updated ?